test/unit/ui/camera.test.js

```diff
@@ -670,15 +670,13 @@ test('camera', (t) => {
             t.end();
         });
 
-        // eslint-disable-next-line no-warning-comments
-        // FIXME(globe-view)
-        // t.test('zooms around a point', (t) => {
-        //     const camera = createCamera();
-        //     camera.easeTo({around: [100, 0], zoom: 3, duration: 0});
-        //     t.deepEqual(fixedLngLat(camera.getCenter()), fixedLngLat({lng: 87.5, lat: 0}));
-        //     t.equal(camera.getZoom(), 3);
-        //     t.end();
-        // });
+        t.test('zooms around a point', (t) => {
```

Comment:
Now addressed as the globe code path is disabled [here](https://github.com/mapbox/mapbox-gl-js/pull/11264/files#diff-6e31e9c70d2c897b3a306e5a6718a4b6698e8b5eb3a11729da97fe21ccc45261L1156).